specs/online_order_spec.rb

```diff
@@ -55,17 +73,41 @@
       # TODO: Your test code here!
       # Useful checks might include:
       #   - OnlineOrder.all returns an array
+      it "Returns an array object" do
+        online_orders = Grocery::OnlineOrder.all
+        online_orders.must_be_kind_of Array #or Array < Object
+      end
       #   - Everything in the array is an Order
+      it "Everything in the array is an Order" do
+        online_orders = Grocery::OnlineOrder.all
+        online_orders.must_be_kind_of Grocery::Order
```

Comment:
If you're testing that everything in the array is an order, you might want a test that looks like this instead:
```
it "Everything in the array is an Order" do
        online_orders = Grocery::OnlineOrder.all
        online_orders.each do |order|
          order.must_be_kind_of Grocery::Order
        end
      end
```